lib/internal/streams/utils.js

```diff
@@ -261,7 +275,23 @@ function isErrored(stream) {
   ));
 }
 
+function encodeWithFastPath(input, encoding) {
+  const enc = normalizeEncoding(encoding);
+
+  if (enc === 'utf8' && TypedArrayPrototypeGetByteLength(input) > 512) {
+    const buf = encoder.encode(input);
+    return new FastBuffer(
+      TypedArrayPrototypeGetBuffer(buf),
+      TypedArrayPrototypeGetByteOffset(buf),
+      TypedArrayPrototypeGetByteLength(buf),
```

Comment:
nit
```suggestion
  const byteLength = TypedArrayPrototypeGetByteLength(input);
  if (enc === 'utf8' && byteLength > 512) {
    const buf = encoder.encode(input);
    return new FastBuffer(
      TypedArrayPrototypeGetBuffer(buf),
      TypedArrayPrototypeGetByteOffset(buf),
      byteLength,
```

Also the benchmark run is failing now, for some reason.

Reply:
Thanks for the review, and feedback. I've updated the code according to your suggestion.